Makefile

```diff
@@ -218,7 +218,7 @@ $(GRPC_DIR):
 # If you want to force update/sync, invoke 'make submodule-sync' directly
 submodule-sync:
 	git submodule sync
-	git submodule update --init
+	git submodule update --init --recursive --remote
```

Comment:
I wasn't aware of --remote but looking at the doc I don't think it would make the fork sha available/work would it ?

so yes please revert for now